### DISCUSSION TITLE
Android Java Scene Viewer, how to access a GLB file in a server with basic authentication?

### DISCUSSION THREAD

**Discussion by rfcj** [google/model-viewer#2742] [Q&A]
I need do use Scene Viewer and access a GLB file in a server with Basic Authentication. I try: file=https://user:password@www.server.com/files/000005.glb but the object don't load. I also try file=https://www.server.com/getfile.php and getfile.php redirect to https://user:password@www.server.com/files/000005.glb but the object don't load. Is there any way?

**Comment by elalish** [comment]
I don't think that's possible with Scene Viewer due to the security boundary of leaving the browser: https://github.com/google/model-viewer/discussions/2187#discussioncomment-558914

Will WebXR work for your needs? That stays in  the browser, which avoids this problem entirely.

**Reply by rfcj** [reply]
I am using the scene viewer in an Android App. I try to download the glb file to Android Download directory and access but don't work.

**Reply by elalish** [reply]
@tpsiaki Any ideas?
